src/hooks/useDisableScroll.tsx

```diff
@@ -0,0 +1,13 @@
+import { useEffect } from "react";
+
+const useDisableScroll = (open: boolean) => {
+  useEffect(() => {
+    if (open) {
+      document.body.style.overflow = "hidden";
+    } else {
+      document.body.style.overflow = "unset";
+    }
+  }, [open]);
```

Comment:
```suggestion
const useDisableScroll = (open: boolean) => {
  useEffect(() => {
    if (open) {
      document.body.style.overflow = "hidden";
    } else {
      document.body.style.overflow = "unset";
    }
    return () => document.body.style.overflow = "unset"
  }, [open]);
```

클린업 함수 있어야 하지 않을까요??

Reply:
이벤트를 등록한게 아니라서 필요없지 않을까요?

Reply:
모달 창에서 버튼을 달아 다른 페이지로 이동한다던가, 뒤로가기와 같은 이벤트 경우를 위해 클린업 함수 필요하지 않을까요??? @imYourChoi

Reply:
아아 그 생각을 못했네요. 수정하겠습니다~

Reply:
감사합니당~~